awx/api/serializers.py

```diff
@@ -5491,7 +5491,7 @@ def get_field_from_model_or_attrs(fd):
         if peers_from_control_nodes and node_type not in (Instance.Types.EXECUTION, Instance.Types.HOP):
             raise serializers.ValidationError(_("peers_from_control_nodes can only be enabled for execution or hop nodes."))
 
-        if node_type in (Instance.Types.CONTROL):
+        if node_type == (Instance.Types.CONTROL):
```

Comment:
I am guessing one of the linters would catch this, but:

```suggestion
        if node_type == Instance.Types.CONTROL:
```